lib/internal/simulation_manager_mixins/peripheral_scanning_mixin.dart

```diff
@@ -5,15 +5,16 @@ mixin PeripheralScanningMixing on SimulationManagerBase {
 
   Future<void> _startDeviceScan() async {
     _peripherals.values.forEach((peripheral) {
-      _scanSubscriptions.add(
-          peripheral.onScan(allowDuplicates: true).listen((scanResult) async {
-        await _bridge.publishScanResult(scanResult);
+      _scanSubscriptions
+          .add(peripheral.onScan(allowDuplicates: true).listen((scanResult) {
```

Comment:
Should `true` be hardcoded?

Reply:
In this case yes, because I think we've misinterpreted the meaning of this in the BLEmulator internals. Anyway: I haven't changed any mechanics, this is the same code that was running before. Those two functions have been only moved from SimulationManager to the mixin.